src/additional/accept.js

```diff
@@ -14,11 +15,23 @@ $.validator.addMethod("accept", function(value, element, param) {
 		// If we are using a wildcard, make it regex friendly
 		typeParam = typeParam.replace(/\*/g, ".*");
 
+		// Allow checking for BLANK mimetype.
+		if (typeParam.match(/^\|/) || typeParam.match(/\|\|/)) {
```

Comment:
Just do another commit at the same branch. We can squash it later when everything looks fine.

Could you also add a few unit tests?